submissions/OlegPopovych/html_css_popup/index.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link href="https://fonts.googleapis.com/css?family=Lato:100,300,400,700,900" rel="stylesheet">
+	<link rel="stylesheet" href="css/main.css">
+	<link rel="shortcut icon" type="image/png" href="img/favicon.png">
+	<title>Popup</title>
+</head>
+
+<body>
+	<nav class="nav">
+		<ul class="nav__list">
+			<li class=" nav__list--item">
+				<h1 class="nav__title">Hello Kottans!</h1>
+			</li>
+			<li class="nav__list--item">
+				<a class="nav__link" href="#">
+					<span>Mail</span>
```

Comment:
It is not necessary to wrap text in `span` inside `a` tag

Reply:
Fixed it